community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/input/Groups.java

```diff
@@ -38,6 +38,11 @@
     private final List<Group> byId = new ArrayList<>( asList( Group.GLOBAL ) );
     private int nextId = LOWEST_NONGLOBAL_ID;
 
+    public Groups()
+    {
+
+    }
```

Comment:
Whaa?

Reply:
Haha, this was a constructor usage trick I pulled for something completely different... I forgot to remove it.